components/dictionary.txt

```diff
@@ -33,6 +34,7 @@ Genomics
 github
 GRCh
 hashedDrops
+HDF
```

Comment:
Just curiousity, but do we ever use bare `HDF` and what happens if we put `HDF5` in the dirctionary?